orm/models.py

```diff
@@ -403,7 +403,7 @@ async def prev(self):
         return await self.query.paginate(self.page - 1, self.per_page)
 
     async def iterate(self):
-        yield await self.prev()
+        yield self
```

Comment:
When we create the pagination query, it already returns the first page requested. Therefore if we want to iterate through all pages we must return again the first page, and then start looping over the remaining pages.